packages/tools/webpack/extract-i18n-expressions-webpack-plugin.js

```diff
@@ -91,16 +100,15 @@ module.exports = class ExtractI18nExpressionsWebpackPlugin {
 				compilation.getPath( '[file]', { filename: chunkJSFile } )
 			);
 
-			// Create Webpack source object which represents a new file, and add it to Webpack assets array.
-			compilation.assets[ assetFilename ] = new RawSource(
-				[ ...( translationCallExpressions.get( mainFilePath ) || new Set() ) ]
-					.map( ( expr ) => `${ expr };` )
-					.join( '' )
-			);
+			const isComment = ( expr ) => /^(\/\/|\/\*).+/.test( expr );
```

Comment:
maybe move it out of the loop?
and why don't just reuse the regexes above? (or maybe they're heavier to run?)

Reply:
Changed